doc/source/advanced.rst

```diff
@@ -260,6 +272,23 @@ Passing a list of labels or tuples works similar to reindexing:
 
    df.loc[[('bar', 'two'), ('qux', 'one')]]
 
+.. warning::
+
+   It is important to note that tuples and lists are not treated identically
+   in pandas. Whereas a tuple is interpreted as one multi-level key, a list is
+   used to specify several keys.
+
+Importantly, a list of tuples indexes several complete ``MultiIndex`` keys,
+whereas a tuple of lists refer to several values within a level:
```

Comment:
I think we need to explain this a bit more in detail

Reply:
Yeah, it could also be just an info box. It is more an advanced example than something you should really know.

Reply:
I'm OK with an info box, but I wish I had really known this in my analysis (which I think isn't that advanced). Anyway, any kind of box is good to attract attention.

Reply:
I don't know whether it is "serious" enough to be written in the docs, but when I happen to discuss this in talks, I always say _"tuples go horizontally [traversing levels], lists go vertically [scanning levels]"_.

Reply:
Done